### DISCUSSION TITLE
Make sender field more visible in compose view

### DISCUSSION THREAD

**Discussion by alstr** [tutao/tutanota#2958] [Ideas]
I don't want to create an issue for this and clog that area up more, but I wanted to see what other people's thoughts were on the compose UI.

I feel it's almost too minimalist. Maybe it's just me, but because the 'sender' address is not visible by default, I've lost track of the number of emails I've sent from the wrong account when quickly writing a message. It would be great if that could be more visible. I do like simplicity though.

**Comment by charlag** [comment]
It would be great to improve this somewhow but might be a bit tricky

**Comment by jowlo** [comment]
So we have an issue for this #3061 and will change on how it is displayed in the next release as this has been requested quite often. 

Please let us know what you think of it (after the release)!

**Reply by alstr** [reply]
That's good to hear, I look forward to trying it. Thanks.
